chromadb/db/__init__.py

```diff
@@ -120,7 +119,7 @@ def get_nearest_neighbors(
         embeddings: Optional[Embeddings] = None,
         n_results: int = 10,
         where_document: WhereDocument = {},
-    ) -> Tuple[List[List[UUID]], npt.NDArray]:
+    ) -> Tuple[List[List[UUID]], List[List[float]]]:
```

Comment:
what's funny here is `HSNW:get_nearest_neighbors` already has a return type of ` List[List[float]]`, not `npt.NDArray`